docs/build/reference/dynamic-deopt.md

```diff
@@ -1,11 +1,12 @@
 ---
 title: "/dynamicdeopt (Enable C++ Dynamic Debugging (Preview))"
-description: "Use the Microsoft C++ compiler option /dynamicdeopt to use C++ Dynamic Debugging."
+description: "Enable the Microsoft C++ compiler option /dynamicdeopt to use C++ Dynamic Debugging."
 ms.date: 03/14/2025
 f1_keywords: ["/dynamicdeopt", "VC.Project.VCNMakeTool.CompileAsManaged", "VC.Project.VCCLCompilerTool.CompileAsManaged"]
 helpviewer_keywords: ["cl.exe compiler, common language runtime option", "-dynamicdeopt compiler option [C++]", "dynamicdeopt compiler option [C++]", "/clr compiler option [C++]", "Managed Extensions for C++, compiling", "common language runtime, /clr compiler option"]
```

Comment:
Just want to check if the `CompileAsManaged` F1 keywords and CLR related helpviewer keywords are correct? Same for the link in the `Set this compiler option programmatically` section.

Reply:
keywords are not correct. I'll fix those via your PR.